exercise2/model/calModel.php

```diff
@@ -0,0 +1,15 @@
+<?php 
+include 'controller/calController.php';
```

Comment:
File names like class names, MUST be declared in StudlyCaps. Also, file names should be the same with its class name for best practices.